embed/web-extension/ephy-web-extension.c

```diff
@@ -368,9 +361,8 @@ request_decision_on_storing (EphyEmbedFormAuth *form_auth)
 }
 
 static void
-should_store_cb (const char *username,
-                 const char *password,
-                 gpointer    user_data)
+should_store_cb (GSList   *records,
```

Comment:
@mcatanzaro That's because this callback returns the result of `ephy_password_manager_query()`. And a query can return more than one items, depending on the query parameters (e.g. if you skip the username, then the query will return the passwords for all the users). However, in this case, all the query parameters are present, so the returned list contains only one record. I know this may seem confusing, but I thought having a generic query function is better than writing a separate function for this scenario only.